assets/scss/partials/components/_wrapper.scss

```diff
@@ -13,12 +13,23 @@
   &__main {
     width: 100%;
 
+    &--sidebarhidden {
```

Comment:
I'd put this modifier not in context of the sidebar but when `main` is allowed to expand to fullscreen. It could even be `--fullscreen`.